docker/ego-init/init.sql

```diff
@@ -295,6 +295,7 @@ ALTER TABLE public.userpermission OWNER TO postgres;
 --
 
 COPY public.egoapplication (name, clientid, clientsecret, redirecturi, description, status, id, type) FROM stdin;
+oauth2-resource	oauth	oauthsecret	http://example.com	song	APPROVED	77f1ef78-7495-4b4a-982a-6b9532dc69fb	CLIENT
```

Comment:
It looks like the application token comming out of ego has `aud` calims in it 
```
{
  "iat": 1586974587,
  "exp": 1586985387,
  "aud": [
    "ego"
  ],
```
The Spring Oauth2 library in score currently doesn't have an explicit check for this claim so it default to the Spring one. This by default is `oauth2-resource` so any aud claim without this is rejected. Depending on how we want to do things, either score needs to configure expected aud via [ResorceServerConfigurer](https://docs.spring.io/spring-security/oauth/apidocs/org/springframework/security/oauth2/config/annotation/web/configurers/ResourceServerSecurityConfigurer.html#resourceId(java.lang.String)) or maybe ignore the claim or maybe ego needs to make sure this claim isn't their.

Reply:
I've opened the PR anyways since I was able to work around the issue to implement the feature, and its behind a profile so won't work until turned on. But, I'm open to leaving it blocked though if we prefer to wait  until those work is done.

Reply:
this is no longer an issue from ego since we removed applcation `aud` claims in https://github.com/overture-stack/ego/issues/513